<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="7" hostname="IZDT-211" name="com.r1v2.backend.pagebuilder.module.ScenarioContentPage" tests="8" failures="0" timestamp="06 Aug 2018 09:56:57 GMT" time="2536.524" errors="1">
  <testcase name="beforeClass" time="2536.524" classname="com.r1v2.backend.pagebuilder.module.ScenarioContentPage">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.r1v2.common.BaseTest.initWebDriver(BaseTest.java:34)
at com.r1v2.common.BaseTest.beforeClass(BaseTest.java:22)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- beforeClass -->
  <testcase name="testPG_1" time="0.000" classname="com.r1v2.backend.pagebuilder.module.ScenarioContentPage">
    <skipped/>
  </testcase> <!-- testPG_1 -->
  <testcase name="testPG_2" time="0.000" classname="com.r1v2.backend.pagebuilder.module.ScenarioContentPage">
    <skipped/>
  </testcase> <!-- testPG_2 -->
  <testcase name="testPG_3" time="0.000" classname="com.r1v2.backend.pagebuilder.module.ScenarioContentPage">
    <skipped/>
  </testcase> <!-- testPG_3 -->
  <testcase name="testPG_4" time="0.000" classname="com.r1v2.backend.pagebuilder.module.ScenarioContentPage">
    <skipped/>
  </testcase> <!-- testPG_4 -->
  <testcase name="testPG_5" time="0.000" classname="com.r1v2.backend.pagebuilder.module.ScenarioContentPage">
    <skipped/>
  </testcase> <!-- testPG_5 -->
  <testcase name="testPG_6" time="0.000" classname="com.r1v2.backend.pagebuilder.module.ScenarioContentPage">
    <skipped/>
  </testcase> <!-- testPG_6 -->
  <testcase name="testPG_7" time="0.000" classname="com.r1v2.backend.pagebuilder.module.ScenarioContentPage">
    <skipped/>
  </testcase> <!-- testPG_7 -->
</testsuite> <!-- com.r1v2.backend.pagebuilder.module.ScenarioContentPage -->
